python/arcticdb/adapters/azure_library_adapter.py

```diff
@@ -26,7 +28,14 @@
 @dataclass
 class ParsedQuery:
     Path_prefix: Optional[str] = None
-    CA_cert_path: str = ""
+    # winhttp is used as Azure backend support on Winodws by default; winhttp itself mainatains ca cert.
+    # The options should be left empty else libcurl will be used on Windows
+    CA_cert_path: str = (
+        ssl.get_default_verify_paths().cafile if platform.system() is not "Windows" else ""
```

Comment:
(nit/personal preference) I don't like the double-negative in these expressions, prefer

```suggestion
        "" if platform.system() is "Windows" else ssl.get_default_verify_paths().cafile
```